sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -1469,6 +1493,30 @@ def hpat_pandas_series_ne_impl(self, other):
 @overload_method(SeriesType, 'add')
 def hpat_pandas_series_add(self, other, level=None, fill_value=None, axis=0):
     """
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
+
+    Pandas API: pandas.Series.add
+
+    Examples
+    --------
+    .. literalinclude:: ../../../examples/series/add.py
+       :language: python
+       :lines: 27-
+       :caption: Getting the addition of series and other
+       :name: ex_series_add
+
+    .. code-block:: console
+
+        > python ./series/add.py
+        a    4.0
+        b    3.0
+        c    NaN
+        d    NaN
+        dtype: float64
+
+    Intel Scalable Dataframe Compiler Developer Guide
```

Comment:
Missing See Also. Pandas API reference has See Also section referring to radd

Reply:
done